packages/contentful/package.json

```diff
@@ -11,11 +11,12 @@
   "scripts": {
     "build:babel": "../../scripts/build-babel.sh build",
     "watch:babel": "../../scripts/build-babel.sh watch",
-    "space:export:crn": "contentful space export --skip-content=true --space-id=$CONTENTFUL_SPACE_ID --environment-id=$CONTENTFUL_ENVIRONMENT --content-file=./schema/crn-schema.json",
-    "space:import:crn": "contentful space import --content-model-only=true --space-id=$CONTENTFUL_SPACE_ID --environment-id=$CONTENTFUL_ENVIRONMENT --content-file=./schema/crn-schema.json",
+    "space:migrate:crn": "yarn ctf-migrate up --all",
```

Comment:
this is fine but it will cause the github actions to fail on master - see:
https://github.com/yldio/asap-hub/blob/af464a8f1d30ea0ef072ad6683ac7d3690f5f106/.github/workflows/reusable-deployment.yml#L218